tests/scripts/test_stdlib.py

```diff
@@ -24,6 +24,20 @@ def test_check_multiline_output_no_split():
     assert run(a_command, split=False)['stdout'] == u'This a multi-\nline No Split test!\n'
 
 
+def test_check_error():
+    a_command = ['false']
+    try:
```

Comment:
Please use pytest.raises for testing exception is raised, smth like 

```
with pytest.raises(CalledProcessError) as err:                                                  
    run(a_command, checked=True) 
```

https://docs.pytest.org/en/latest/assert.html

Reply:
Ok. Changed in the current patch.